requirements-dev.txt

```diff
@@ -14,7 +14,7 @@ ansible-core==2.16.3
     #   ansible-compat
     #   ansible-lint
     #   molecule
-ansible-lint==6.22.2
+ansible-lint==24.2.0
```

Comment:
They changed the versioning scheme, so `24.2.0` is actually the successor of `6.22.2`.